sdk/core/src/_helpers.ts

```diff
@@ -1,6 +1,9 @@
 import { SUPPORTED_DID_METHODS } from './_defaultConfig'
 import SdkErrorFromCode from './shared/SdkErrorFromCode'
 
+import { version } from '../package.json'
```

Comment:
Do we know that it will work in browser, after everything was minified and bundled etc?

Reply:
Hm, tried that on top of https://github.com/affinityproject/affinidi-demo-browser-app with bumping up sdk version and that puts header correctly, but haven't tried running on minified and bundled version. Will update that here.

Reply:
that is really didn't work in bundle and seems because of that `bootstrap` stage failed in CodeClimate.
Changed to `require('../package.json')`.